L04-generics/src/main/java/homework/CustomerReverseOrder.java

```diff
@@ -0,0 +1,20 @@
+package homework;
+
+import java.util.LinkedList;
+
+public class CustomerReverseOrder {
+
+    private final LinkedList<Customer> stack;
```

Comment:
Можно сузить интерфейс:

```suggestion
    private final Deque<Customer> stack;
```